src/containers/VisibleMovieDetails/index.jsx

```diff
@@ -1,14 +1,19 @@
-import React, { useState, useEffect } from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 
 import MovieDetails from '../../components/Header/MovieDetails';
-import * as data from '../../data';
+import { currentMovieIdSelector } from '../../slices/moviesInfo';
+import { fetchMovie, movieSelector } from '../../slices/movieInfo';
 
 const VisibleMovieDetails = () => {
-  const [movie, setMovie] = useState({});
+  const dispatch = useDispatch();
+
+  const id = useSelector(currentMovieIdSelector);
```

Comment:
It doesn't make sense to use the API to receive movie details here as the movies list contains all the necessary information. Consider using **movieByIdSelector** there